### PR TITLE
addition of contributor jpuple

### PR DESCRIPTION
**Pull request by J-Purple**

### CODE REVIEW

**Comment by RMPR** [comment]
Minor nitpick, there's a typo in your commit message
> jpuple

File: content/post/contributors/jpurple.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Jpurple"
+date: 2021-12-13T18:10:23+01:00
+images: []
+categories: []
+tags: []
+authors: ["Jpurple"]
+---
+
+Hi there, I'm Ngnepedoh Irving Craig( yh but call me purple/jpurple ), young aspiring developper. I am an IT enthusiast and am fascinated by Artificial Intelligence. I am familiar with Java, Python and C. Currently working as an android app developper. I look foward to working with you all and am excited about what I will learn here.
```
Comment (RMPR) [inline content/post/contributors/jpurple.md:10]:
```suggestion
Hi there, I'm Ngnepedoh Irving Craig (yh but call me purple/jpurple), young aspiring developer. I am an IT enthusiast and am fascinated by Artificial Intelligence. I am familiar with Java, Python and C. Currently working as an android app developer. I look forward to working with you all and am excited about what I will learn here.
```

**Review by RMPR — CHANGES_REQUESTED** [review]

**Comment by J-Purple** [comment]
oh thanks, just committed the changes...

**Review by elhmn — APPROVED** [review]

**Review by RMPR — APPROVED** [review]

**Comment by RMPR** [comment]
@elhmn you can add yotas to this one as well

**Pull request merged**